docs/workloads/common/data/pysparkle/pysparkle_quickstart.md

```diff
@@ -13,32 +13,26 @@ keywords:
   - etl
 ---
 
-[PySparkle](https://github.com/amido/stacks-azure-data/tree/main/pysparkle) is a Python library
-built to streamline data processing workflows with Apache Spark.
+[PySparkle](https://github.com/amido/stacks-azure-data/tree/main/pysparkle) is a Python library built to streamline
+data processing workflows with Apache Spark.
 
-As an integral component of the Stacks Data Platform, PySparkle is our example of data transformation
-in action. It's designed to showcase the capabilities of our platform. The application operates
-on [The Movies Dataset](https://www.kaggle.com/datasets/rounakbanik/the-movies-dataset) available
-on Kaggle, which contains metadata and ratings for 45,000 movies released on or before July 2017.
-
-**The Movies Dataset** contains a number of CSV files and includes values of different types,
-including nested datatypes and arrays, which give us enough complexity for transformations.
+As an integral component of the Stacks Data Platform, PySparkle is our example of data transformation in action.
+It's designed to showcase the capabilities of our platform.
```

Comment:
Following the suggestion above, would delete this.

Reply:
done